### PR TITLE
[feature] use of Adoptiums 'temurin' JDK

### PR DESCRIPTION
**Pull request by reinhapa**
### Description:
Uses the Eclipse Foundation Adoptium JDK implementation for the build pipeline.

### CODE REVIEW

**Comment by adamretter** [comment]
Hi @reinhapa this looks interesting. What is the difference between `temurin` and what we were using before?

**Comment by reinhapa** [comment]
@adamretter  as far I was seen it the default is Azul's Zulu at the moment.

**Comment by adamretter** [comment]
Yes I think we use Azul at the mo, but what is temurin compared to openjdk plain?

**Comment by dizzzz** [comment]
seen this? https://adoptium.net

**Comment by line-o** [comment]
Related #4084

**Comment by line-o** [comment]
The same issue on Windows surfaces that I already noticed in #4084 
--> https://github.com/eXist-db/exist/runs/4291841020?check_suite_focus=true#step:5:11411

**Comment by line-o** [comment]
#4084 also uses built-in caching of maven artifacts.

**Comment by line-o** [comment]
Cool! It seems to work.

File: exist-parent/pom.xml [hunk 1]
```diff
@@ -104,7 +104,7 @@
         <apache.httpcomponents.version>4.5.13</apache.httpcomponents.version>
         <apache.httpcomponents.core.version>4.4.14</apache.httpcomponents.core.version>
         <apache.xmlrpc.version>3.1.3</apache.xmlrpc.version>
-        <aspectj.version>1.9.4</aspectj.version>
+        <aspectj.version>1.9.8.M1</aspectj.version>
```
Comment (adamretter) [inline exist-parent/pom.xml:107]: @reinhapa I guess you saw this one - https://bugs.eclipse.org/bugs/show_bug.cgi?id=553623  ?

**Comment by dizzzz** [comment]
image

**Review by dizzzz — APPROVED** [review]

File: exist-parent/pom.xml (hunk 1 above)
Comment (line-o) [reply exist-parent/pom.xml:107]: Wow, two years. @reinhapa might have picked up my workaround from
https://github.com/eXist-db/exist/pull/4084/commits/fe77cd696da550e3709fa01136eb415bc9b21360

**Review by line-o — APPROVED** [review]

**Comment by dizzzz** [comment]
OK to go?

**Comment by line-o** [comment]
@dizzzz I would like to restart the jobs once to see if JDK8 on windows-latest also runs through.

**Comment by dizzzz** [comment]
@line-o  restarted

**Comment by dizzzz** [comment]
yeas!!!!!!

**Pull request merged**